hosted/eks/helper/helper_cluster.go

```diff
@@ -572,44 +581,22 @@ func DeleteEKSClusterOnAWS(region string, clusterName string) error {
 
 // <==============================EKS CLI(end)==============================>
 
-// defaultEKS returns a version less than the highest version or K8S_UPGRADE_MINOR_VERSION if it is set.
-// Note: It does not return the default version used by UI which is the highest supported version.
-func defaultEKS(client *rancher.Client, forUpgrade bool) (defaultEKS string, err error) {
-
-	var allVersions []string
-	allVersions, err = kubernetesversions.ListEKSAllVersions(client)
-	if err != nil {
-		return
-	}
-
-	versions := helpers.FilterUIUnsupportedVersions(allVersions, client)
-	maxValue := helpers.HighestK8sMinorVersionSupportedByUI(client)
-
-	for i := 0; i < len(versions); i++ {
-		version := versions[i]
-		// If UI maxValue not yet supported by operator
-		if !strings.Contains(version, maxValue) {
-			maxValue = versions[0]
-		}
-
-		if forUpgrade {
-			if result := helpers.VersionCompare(version, maxValue); result == -1 {
-				return version, nil
-			}
-		} else {
-			if strings.Contains(version, maxValue) {
-				return version, nil
-			}
-		}
-	}
-	return
-}
-
 // GetK8sVersion returns the k8s version to be used by the test;
 // this value can either be envvar DOWNSTREAM_K8S_MINOR_VERSION or the default UI value returned by DefaultEKS.
 func GetK8sVersion(client *rancher.Client, forUpgrade bool) (string, error) {
 	if k8sVersion := helpers.DownstreamK8sMinorVersion; k8sVersion != "" {
 		return k8sVersion, nil
 	}
-	return defaultEKS(client, forUpgrade)
+	allVariants, err := ListEKSAllVersions(client)
+	if err != nil {
+		return "", err
+	}
+
+	if forUpgrade {
```

Comment:
These could be a common helper function

Reply:
Do you mean the entire `GetK8sVersion` can be moved to helper_common?